test/loco/integer/magicseries.clj

```diff
@@ -0,0 +1,75 @@
+(ns ^{:author "Andrea Richiardi"
+      :doc "Magic Series example + tests.
+            A series S = (S0,...Sn) is magic if Si represents the number of occurrences of i in S.
+            Example: A Magic Series of lenght 4 is |2|1|2|0|0|"}
+  loco.integer.magicseries
+  (:require [clojure.test :refer :all]
+            [loco.core :refer :all]
+            [loco.constraints :refer :all]
+            [criterium.core :refer [bench]]))
+
+(defn series-vars
+  "Returns the control variables of the Magic Series."
+  [series-lenght]
+  (for [i (range series-lenght)]
```

Comment:
You seem to be consistently misspelling this, shouldn't it be `series-length`? :)